src/server/sql/migration/get_all_migrations.sql

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+-- get the current as max of to_version column
```

Comment:
This comment does not make sense in respect to `SELECT * FROM migrations`. Can you elaborate?